src/instrumentsscene/view/instrumentsettingsmodel.cpp

```diff
@@ -103,6 +104,30 @@ void InstrumentSettingsModel::resetAllFormatting()
     notationParts()->replacePart(m_instrumentKey.partId, masterPart->clone());
 }
 
+void InstrumentSettingsModel::cloneInstrument()
+{
+    if (!notationParts()) {
+        return;
+    }
+
+    Part* part = notationParts()->part(m_instrumentKey.partId)->clone();
+
+    if (!part) {
+        return;
+    }
+
+    part->setPartName(part->partName() + String(" (copy)"));
```

Comment:
You need to take translatability into account

Reply:
You won't be able to do that, right? You can make " (copy)" translatable. But won't different languages have different sentence structures.

But anyway, I will make it translatable since its already being done for when you duplicate a part in the Parts dialog